proto/ibc/applications/transfer/v1/authz.proto

```diff
@@ -23,9 +23,17 @@ message Allocation {
   // allow list of memo strings, an empty list prohibits all memo strings;
   // a list only with "*" permits any memo string
   repeated string allowed_packet_data = 5;
+  // Forwarding options that are allowed.
+  AllowedForwarding allowed_forwarding = 6 [(gogoproto.nullable) = false];
+}
+
+// AllowedForwarding defines which options are allowed for forwarding.
+message AllowedForwarding {
   // a list of allowed source port ID/channel ID pairs through which the packet is allowed to be forwarded until final
   // destination
-  repeated Hops allowed_forwarding_hops = 6 [(gogoproto.nullable) = false];
+  repeated Hops allowed_hops = 1 [(gogoproto.nullable) = false];
+  // Whether to allow automatic unwinding of tokens to the original chain.
+  bool allow_unwind = 2;
```

Comment:
I would switch the fields around to [match the order in `Forwarding` message for `MsgTransfer`](https://github.com/cosmos/ibc-go/blob/6619821c739891ff132e42eb1c1a49f5e0791e7c/proto/ibc/applications/transfer/v1/transfer.proto#L25-L30)